facturacion-perinatal/index.ts

```diff
@@ -14,12 +14,20 @@ router.group('/perinatalSql', (group) => {
     group.post('/create', async (req: any, res) => {
         res.send({ message: 'ok' });
         try {
-            const prestacion = req.body.data;
-            // verificamos si la prestacion es "Consulta de control de embarazo"
-            const conceptConsultaEmbarazo = ['1191000013107', '721000013105', '1181000013109', '1141000013103',
+            const prestacion = req.body.data.prestacion;
+            const conceptConsultaEmbarazo = ['1191000013107', '1141000013103',
                 '1341000013100', '1201000013105', '1741000013104', '4501000013105', '1711000013103'];
             const conceptId = prestacion.solicitud.tipoPrestacion.conceptId;
-            if (conceptConsultaEmbarazo.includes(conceptId)) {
+            // verificamos si la prestación es algunas de las relacionadas a control de embarazo
+            // si no lo es, entonces verificamos que contenga entre sus registros alguno estos conceptos
+            let enviarFacturacion: boolean = conceptConsultaEmbarazo.includes(conceptId);
+
+            const registros = getRegistros(prestacion.ejecucion.registros);
+            if (!enviarFacturacion) {
+                // Cuando se registra concepto número de "embarazo" de cualquier otra prestación.
+                enviarFacturacion = registros.find(reg => conceptConsultaEmbarazo.includes(reg.conceptId));
+            }
```

Comment:
Consultar los registros solo en caso de precisarlos
```suggestion
            if (!enviarFacturacion) {
                // Cuando se registra concepto número de "embarazo" de cualquier otra prestación.
                const registros = getRegistros(prestacion.ejecucion.registros);
                enviarFacturacion = registros.find(reg => conceptConsultaEmbarazo.includes(reg.conceptId));
            }
```